MapboxCoreNavigation/NavigationSettings.swift

```diff
@@ -15,7 +15,7 @@ extension Notification.Name {
  Properties are prefixed and before they are stored in UserDefaults.standard.
  */
 @objc(MBNavigationSettings)
-public class NavigationSettings: NSObject {
+open class NavigationSettings: NSObject {
```

Comment:
What are the implications for the singleton `NavigationSettings.shared`? Is it even possible to use an instance of a NavigationSettings subclass?

Reply:
I figured e.g overriding `voiceMuted`’s didSet could be useful for changing a custom mute icon but a developer can also listen for `.navigationSettingsDidChange` notifications or KVO `.shared.observe(./voiceMuted …`. There are some bad implications though for subclassing this singleton, duplicated notifications being the first that comes to mind so let's avoid subclassing this class by reverting back to public.